src/main/java/frc/trigon/robot/subsystems/arm/ArmConstants.java

```diff
@@ -0,0 +1,62 @@
+package frc.trigon.robot.subsystems.arm;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.trajectory.TrapezoidProfile;
+import edu.wpi.first.wpilibj.smartdashboard.Mechanism2d;
+import edu.wpi.first.wpilibj.smartdashboard.MechanismLigament2d;
+import edu.wpi.first.wpilibj.smartdashboard.MechanismRoot2d;
+import edu.wpi.first.wpilibj.util.Color;
+import edu.wpi.first.wpilibj.util.Color8Bit;
+import org.littletonrobotics.junction.AutoLogOutput;
+
+public class ArmConstants {
+    private static final double
+            MAX_ANGLE_VELOCITY = 150,
+            MAX_ANGLE_ACCELERATION = 190,
+            MAX_ELEVATOR_VELOCITY = 2,
+            MAX_ELEVATOR_ACCELERATION = 3;
+    static final TrapezoidProfile.Constraints
+            ANGLE_CONSTRAINTS = new TrapezoidProfile.Constraints(MAX_ANGLE_VELOCITY, MAX_ANGLE_ACCELERATION),
+            ELEVATOR_CONSTRAINTS = new TrapezoidProfile.Constraints(MAX_ELEVATOR_VELOCITY, MAX_ELEVATOR_ACCELERATION);
+
+    static final double
+            ANGLE_TOLERANCE_DEGREES = 0.5,
+            ELEVATOR_TOLERANCE_METERS = 0.02;
+
+    private static final double
+            ARM_MECHANISM_WIDTH_METERS = 3,
+            ARM_MECHANISM_HEIGHT_METERS = 3,
+            LIGAMENT_LINE_WIDTH = 10;
+    public static final double RETRACTED_ARM_LENGTH_METERS = 0.64;
+    private static final double
+            ARM_MECHANISM_ROOT_X = 1,
+            ARM_MECHANISM_ROOT_Y = 1;
+    static final Mechanism2d ARM_MECHANISM = new Mechanism2d(
+            ARM_MECHANISM_WIDTH_METERS,
+            ARM_MECHANISM_HEIGHT_METERS
+    );
+    @AutoLogOutput
```

Comment:
You want to put this above the ARM_MECHANISM variable to log it, and specify the path using the key variable (`@Annotation(variable = value)`)